.github/workflows/draft_pr_comment.yml

```diff
@@ -0,0 +1,22 @@
+name: draftprcommentbot
+
+on:
+  pull_request_target:
+    types: ['opened']
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+    - name: 'Comment Draft PR'
+      uses: actions/github-script@v3
```

Comment:
Maybe add a comment where the script comes from?

Reply:
It is from https://github.com/actions/github-script

Reply:
Thanks for the feedback! I'll refrain from making this too fancy, though, until we have a buy-in. I just want to show that it is possible right now. For a more complicated workflow, like checking if the bot has commented before, I'll have to investigate more and maybe writing a script to go with this action.

Reply:
It certainly gives a sense of what's possible; agreed that we should first discuss what to do, since ideally we do not have two different systems.